its/plugin/projects/hadolint_project/docker/hadolint-report.json

```diff
@@ -0,0 +1,10 @@
+[
```

Comment:
I would suggest using some bigger more real-life example (more issues, bigger input file, different type of issues, etc) in integration tests.